app/src/test/java/org/oppia/app/testing/ImageRegionSelectionInteractionViewTest.kt

```diff
@@ -79,81 +98,103 @@ class ImageRegionSelectionAccessibilityTest {
   }
 
   @Test
-  fun testImageRegionSelectionTestActivity_clickOnRegion3() {
+  fun testImageRegionSelectionInteractionView_clickRegion3_Region2Clicked() {
     launch(ImageRegionSelectionTestActivity::class.java).use {
-      onView(withId(R.id.clickable_image_view)).perform(
-        clickAtXY(0.3f, 0.3f)
-      )
-      onView(allOf(withTagValue(`is`("Region 3"))))
-        .check(
-          matches(isDisplayed())
-        )
-      onView(allOf(withId(R.id.submit_button)))
-        .check(
-          matches(isEnabled())
+      it.onActivity {
+        it.clickable_image_view.setListener(onClickableAreaClickedListener)
+
+        onView(withId(R.id.clickable_image_view)).perform(
+          clickPoint(0.2f, 0.2f)
         )
+
+        verify(onClickableAreaClickedListener).onClickableAreaTouched(capture(regionClickedEvent))
+        assertEquals(NamedRegionClickedEvent("Region 2"), regionClickedEvent.value)
+      }
     }
   }
 
   @Test
-  fun testImageRegionSelectionTestActivity_clickOnRegion3_clickOnRegion2_onlyRegion2IsDisplayed() {
+  fun testImageRegionSelectionInteractionView_clickRegion3_clickRegion2_Region2Clicked() {
```

Comment:
Here, above, & elsewhere: please make sure that the capitalization is consistent:

``testImageRegionSelectionInteractionView_clickRegion3_clickRegion2_region2Clicked``

Reply:
done